### ISSUE TITLE
Add content in Full-Stack roadmap

### ISSUE THREAD

**Issue by yashdev9274** [BuildersChain/Developers-Roadmap#14]
### Description

First create a draft file similar to [Javascript-roadmap.md](https://github.com/BuildersChain/Developers-Roadmap/blob/main/Frontend-Developer/JavaScript/Javascript-roadmap.md) and present it by opening a separate issue and link that respective issue/pr to particular roadmap

### Screenshots

Display roadmap screenshot  here

**Comment by SagarSharma2809** [comment]
Hi, can I work on this issue?

**Comment by yashdev9274** [comment]
yes go ahead ,  before that go through the roadmap draft mentioned above

**Comment by Saikat-Mandal** [comment]
hello @yashdev9274 is the issue solved ?

**Comment by yashdev9274** [comment]
Yes @Saikat-Mandal it is still open you can work on it ,  before that do have a look at code of conduct

**Comment by yashdev9274** [comment]
Hey @Saikat-Mandal and @SagarSharma2809 are you working on it , how far have you guys worked on it.

**Comment by Saikat-Mandal** [comment]
@yashdev9274 contributing docs is giving 404 error

**Comment by yashdev9274** [comment]
> @yashdev9274 Also should i like make a readme.md file like the one you mentioned as the javascript one and enter the textual version ?

only if you are willing to create new roadmap then create readme draft , but if are want to update the existing directory then there is no need to create it

**Comment by Saikat-Mandal** [comment]
Hey @yashdev9274 take a look at this. Is this how i can proceed ?

![Full-stack-developer md - Developers-Roadmap - Visual Studio Code 05-03-2023 23_14_53](https://user-images.githubusercontent.com/75900118/222976938-d77a1510-3267-4eb7-a210-b20c06cbfd06.png)

**Comment by yashdev9274** [comment]
> Hey @yashdev9274 take a look at this. Is this how i can proceed ?
> 
> ![Full-stack-developer md - Developers-Roadmap - Visual Studio Code 05-03-2023 23_14_53](https://user-images.githubusercontent.com/75900118/222976938-d77a1510-3267-4eb7-a210-b20c06cbfd06.png)

Looks good, go ahead with the changes, and for more details on how to shape the required roadmap refer to [Web3 Develoepr Roadmap](https://github.com/BuildersChain/Developers-Roadmap/tree/main/WEB3-Developer)

**Comment by Saikat-Mandal** [comment]
@yashdev9274 yeah got it !

**Comment by Saikat-Mandal** [comment]
@yashdev9274 I made a new PR . I created a new issue and linked the PR to that. Should i link the PR to this issue also ?
